src/operations/processor.py

```diff
@@ -0,0 +1,157 @@
+"""Модуль для обработки банковских операций."""
+import logging
+import re
+from collections import Counter
+from typing import Dict, List, Any, Pattern
+
+from src.logging_config.logger_config import get_utils_logger
+
+# Инициализация логгера
+logger: logging.Logger = get_utils_logger()
+
+
+def search_operations_by_description(data: List[Dict[str, Any]], search: str) -> List[Dict[str, Any]]:
+    """
+    Ищет операции по строке в описании с использованием регулярных выражений.
+
+    Args:
+        data: Список словарей с данными о банковских операциях
+        search: Строка для поиска в описании операций
+
+    Returns:
+        Список словарей с операциями, у которых в описании есть искомая строка
+    """
+    try:
+        logger.debug(f"Поиск операций по строке: '{search}'")
```

Comment:
Логи правильнее писать так:
```suggestion
        logger.debug(f"Поиск операций по строке: '%s'", search)
```
Это быстрее и безопаснее